exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/FragmentWrapper.java

```diff
@@ -49,58 +51,135 @@ public String getId() {
     return String.format("fragment-%s", major.getMajorFragmentId());
   }
 
-  public static final String[] FRAGMENT_OVERVIEW_COLUMNS = {"Major Fragment", "Minor Fragments Reporting",
-    "First Start", "Last Start", "First End", "Last End", "Min Runtime", "Avg Runtime", "Max Runtime", "Last Update",
-    "Last Progress", "Max Peak Memory"};
+  public static final String[] ACTIVE_FRAGMENT_OVERVIEW_COLUMNS = {"Major Fragment", "Minor Fragments Reporting",
+    "First Start", "Last Start", "First End", "Last End", "Min Runtime", "Avg Runtime", "Max Runtime", "% Busy",
+    "Last Update", "Last Progress", "Max Peak Memory"};
+
+  public static final String[] ACTIVE_FRAGMENT_OVERVIEW_COLUMNS_TOOLTIP = {null, "# Minor Fragments Spawned",
+    null, null, null, null, "Shortest duration of a fragment", "Avg duration of a fragment", "Longest duration of a fragment", "%time Fragments were Busy",
+    "Last time a running fragment's status was updated", "Last time we heard from a running fragment", null};
```

Comment:
Should I retain the 'darkblue' text color change for columns, if they all eventually have tooltips??